examples/options/options-trade-options-with-alpaca.ipynb

```diff
@@ -0,0 +1,807 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check account related information\n",
+    "acc = trade_client.get_account()\n",
+    "acc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Check account configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check account configurations\n",
+    "acct_config = trade_client.get_account_configurations()\n",
+    "acct_config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Retrieve list of assets which are options enabled"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can filter assets by `options_enabled` attribute\n",
+    "# Asset object has `options_enabled` attribute if it is options enabled\n",
+    "req = GetAssetsRequest(\n",
+    "  attributes = \"options_enabled\"  \n",
+    ")\n",
+    "assets = trade_client.get_all_assets(req)\n",
+    "assets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6. Get list of options contracts for the given symbol (e.g. SPY)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# In this instance, we retrieve the necessary contracts based on the parameters we've configured.\n",
+    "# specify underlying symbol\n",
+    "\n",
+    "underlying_symbol = \"SPY\"\n",
+    "req = GetOptionContractsRequest(\n",
+    "    underlying_symbol=[underlying_symbol],  \n",
+    "    # specify asset status: active (default)\n",
+    "    status=AssetStatus.ACTIVE,   \n",
+    "    # specify expiration date (specified date + 1 day range)\n",
+    "    expiration_date = None,\n",
+    "    # we can pass a date object or string (YYYY-MM-DD) to specify an expiration date range\n",
+    "    expiration_date_gte = None,                            \n",
+    "    expiration_date_lte = None, \n",
+    "    # specify root symbol\n",
+    "    root_symbol = None,\n",
+    "    # specify option type (ContractType.CALL or ContractType.PUT)\n",
+    "    type = None,  \n",
+    ")\n",
+    "\n",
+    "res = trade_client.get_option_contracts(req)\n",
+    "res"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 7. Get options contract by symbol"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "symbol = res.option_contracts[0].symbol\n",
+    "contract = trade_client.get_option_contract(symbol)\n",
+    "contract"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 8. Get options contract by id"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
```

Comment:
might be best to randomize this in case the user runs the notebook multiple times